### ISSUE TITLE
pycolmap/_core.cpython-310-x86_64-linux-gnu.so error.How to solve this problem

### ISSUE THREAD

**Issue by kaihekaihe** [ByteDance-Seed/Depth-Anything-3#213]
Basic package list:

uv pip list
Package                   Version         Editable project location
------------------------- --------------- ----------------------------------------------------------
addict                    2.4.0
annotated-doc             0.0.4
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     4.12.1
argcomplete               3.6.3
attrs                     25.4.0
blinker                   1.9.0
certifi                   2026.1.4
cfgv                      3.5.0
charset-normalizer        3.4.4
click                     8.3.1
colorama                  0.4.6
configargparse            1.7.1
contourpy                 1.3.2
cuda-bindings             12.9.4
cuda-pathfinder           1.3.3
cycler                    0.12.1
dash                      3.4.0
decorator                 4.4.2
depth-anything-3          0.0.0           /workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main
distlib                   0.4.0
e3nn                      0.5.9
einops                    0.8.2
evo                       1.34.2
exceptiongroup            1.3.1
fastapi                   0.128.0
fastjsonschema            2.21.2
filelock                  3.20.3
flask                     3.1.2
fonttools                 4.61.1
fsspec                    2026.1.0
gsplat                    1.5.3+pt24cu124
h11                       0.16.0
hf-xet                    1.2.0
httpcore                  1.0.9
httpx                     0.28.1
huggingface-hub           1.3.5
identify                  2.6.16
idna                      3.11
imageio                   2.37.2
imageio-ffmpeg            0.6.0
importlib-metadata        8.7.1
itsdangerous              2.2.0
jaxtyping                 0.3.6
jinja2                    3.1.6
joblib                    1.5.3
jsonschema                4.26.0
jsonschema-specifications 2025.9.1
jupyter-core              5.9.1
kiwisolver                1.4.9
lz4                       4.4.5
markdown-it-py            4.0.0
markupsafe                3.0.3
matplotlib                3.10.8
mdurl                     0.1.2
moviepy                   1.0.3
mpmath                    1.3.0
narwhals                  2.15.0
natsort                   8.4.0
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.4.2
ninja                     1.13.0
nodeenv                   1.10.0
numexpr                   2.14.1
numpy                     1.26.4
nvidia-cublas-cu12        12.8.4.1
nvidia-cuda-cupti-cu12    12.8.90
nvidia-cuda-nvrtc-cu12    12.8.93
nvidia-cuda-runtime-cu12  12.8.90
nvidia-cudnn-cu12         9.10.2.21
nvidia-cufft-cu12         11.3.3.83
nvidia-cufile-cu12        1.13.1.3
nvidia-curand-cu12        10.3.9.90
nvidia-cusolver-cu12      11.7.3.90
nvidia-cusparse-cu12      12.5.8.93
nvidia-cusparselt-cu12    0.7.1
nvidia-nccl-cu12          2.27.5
nvidia-nvjitlink-cu12     12.8.93
nvidia-nvshmem-cu12       3.4.5
nvidia-nvtx-cu12          12.8.90
omegaconf                 2.3.0
open3d                    0.19.0
opencv-python             4.11.0.86
opt-einsum                3.4.0
opt-einsum-fx             0.1.4
packaging                 26.0
pandas                    2.3.3
pathspec                  1.0.4
pillow                    12.1.0
pillow-heif               1.2.0
platformdirs              4.5.1
plotly                    6.5.2
plyfile                   1.1.3
pre-commit                4.5.1
proglog                   0.1.12
pybind11                  3.0.1
pycolmap                  3.13.0
pydantic                  2.12.5
pydantic-core             2.41.5
pygments                  2.19.2
pyparsing                 3.3.2
pyquaternion              0.9.9
python-dateutil           2.9.0.post0
pytz                      2025.2
pyyaml                    6.0.3
referencing               0.37.0
requests                  2.32.5
retrying                  1.4.2
rich                      14.3.1
rosbags                   0.11.0
rpds-py                   0.30.0
ruamel-yaml               0.19.1
safetensors               0.7.0
scikit-build-core         0.11.6
scikit-learn              1.7.2
scipy                     1.15.3
seaborn                   0.13.2
setuptools                80.10.2
shellingham               1.5.4
six                       1.17.0
starlette                 0.50.0
sympy                     1.14.0
threadpoolctl             3.6.0
tomli                     2.4.0
torch                     2.10.0
torchvision               0.25.0
tqdm                      4.67.1
traitlets                 5.14.3
trimesh                   4.11.1
triton                    3.6.0
typer                     0.21.1
typer-slim                0.21.1
typing-extensions         4.15.0
typing-inspection         0.4.2
tzdata                    2025.3
urllib3                   2.6.3
uvicorn                   0.40.0
virtualenv                20.36.1
wadler-lindig             0.1.7
werkzeug                  3.1.5
xformers                  0.0.34
zipp                      3.23.0
zstandard                 0.25.0
(base) root@devpod2:/workspace/nvidia-cosmos/Depth-Anything-3-main# 

uv run --python python3.10 python -c "import torch; print(torch.version.cuda); print(torch.backends.cudnn.version()); print(torch.__version__)"
12.8
91002
2.10.0+cu128

2, The error:
  uv run --python python3.10   da3 auto assets/examples/SOH \
    --export-format glb \
    --export-dir ./out \
    --model-dir /workspace/huggingface/hub/depth-anything/DA3NESTED-GIANT-LARGE-1.1 
🔍 Detected input type: IMAGES
📁 Input path: assets/examples/SOH

Processing directory of images...
Found 2 images to process
Export directory './out' already exists.
Do you want to clean it and continue? [y/N]: y
Cleaned export directory: ./out
Loading model from /workspace/huggingface/hub/depth-anything/DA3NESTED-GIANT-LARGE-1.1...
[INFO ] using SwiGLU layer as FFN
[INFO ] using MLP layer as FFN
Loading weights from local directory
Running inference on 2 images...
[INFO ] Processed Images Done taking 0.05663895606994629 seconds. Shape:  torch.Size([2, 3, 280, 504])
*** Aborted at 1769772423 (unix time) try "date -d @1769772423" if you are using GNU date ***
PC: @     0x7fe48836f137 (/workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main/.venv/lib/python3.10/site-packages/nvidia/cublas/lib/libcublas.so.12+0x16f136)
*** SIGFPE (@0x7fe48836f137) received by PID 1404504 (TID 0x7fe5321eb740) from PID 18446744071699886391; stack trace: ***
    @     0x7fe53228fed3 (/usr/lib/x86_64-linux-gnu/libc.so.6+0xa1ed2)
    @     0x7fe2cfdc6868 (/workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main/.venv/lib/python3.10/site-packages/pycolmap/_core.cpython-310-x86_64-linux-gnu.so+0xa26867)
    @     0x7fe532233330 (/usr/lib/x86_64-linux-gnu/libc.so.6+0x4532f)
    @     0x7fe48836f137 (/workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main/.venv/lib/python3.10/site-packages/nvidia/cublas/lib/libcublas.so.12+0x16f136)
    @     0x7fe48847e4b3 (/workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main/.venv/lib/python3.10/site-packages/nvidia/cublas/lib/libcublas.so.12+0x27e4b2)
    @     0x7fe48847efec (/workspace/huggingface/nvidia-cosmos/Depth-Anything-3-main/.venv/lib/python3.10/site-packages/nvidia/cublas/lib/libcublas.so.12+0x27efeb)
    @     0x7fe4883293ae cublasSdot_v2
    @     0x7fe388de437f void at::cuda::blas::dot<float>(cublasContext*, int, float const*, int, float const*, int, float*)
    @     0x7fe388e87f9a at::native::dot_cuda(at::Tensor const&, at::Tensor const&) [clone .localalias]
    @     0x7fe388add890 c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoFunctor_<c10::CompileTimeFunctionPointer<at::Tensor (at::Tensor const&, at::Tensor const&), &at::(anonymous namespace)::(anonymous namespace)::wrapper_CUDA__dot>, at::Tensor, c10::g4
    @     0x7fe3c54ef481 at::_ops::dot::call(at::Tensor const&, at::Tensor const&)
    @     0x7fe3c4471d33 at::autocast::WrapFunction_<(at::autocast::CastPolicy)4, (c10::DeviceType)1, at::Tensor (at::Tensor const&, at::Tensor const&), &at::_ops::dot::call, at::Tensor, c10::guts::typelist::typelist<at::Tensor const&, at::Tensor const&> >::call(at::Tensor const&,4
    @     0x7fe3c433e880 c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoRuntimeFunctor_<at::Tensor (*)(at::Tensor const&, at::Tensor const&), at::Tensor, c10::guts::typelist::typelist<at::Tensor const&, at::Tensor const&> >, at::Tensor (at::Tensor const4
    @     0x7fe3c54ef481 at::_ops::dot::call(at::Tensor const&, at::Tensor const&)
    @     0x7fe3d835e6d2 torch::autograd::THPVariable_dot(_object*, _object*, _object*)
    @     0x5612c5459162 cfunction_call
    @     0x5612c54a02fc _PyEval_EvalFrameDefault
    @     0x5612c5437bd2 _PyFunction_Vectorcall
    @     0x5612c54c4559 call_function
    @     0x5612c54915c1 _PyEval_EvalFrameDefault
    @     0x5612c5437bd2 _PyFunction_Vectorcall
    @     0x5612c5439bd0 method_vectorcall
    @     0x5612c54977c4 _PyEval_EvalFrameDefault
    @     0x5612c5437bd2 _PyFunction_Vectorcall
    @     0x5612c5439c6f method_vectorcall
    @     0x5612c54958f3 _PyEval_EvalFrameDefault
    @     0x5612c5437bd2 _PyFunction_Vectorcall
    @     0x5612c5439c6f method_vectorcall
    @     0x5612c54958f3 _PyEval_EvalFrameDefault
    @     0x5612c5437bd2 _PyFunction_Vectorcall
    @     0x5612c55364a6 _PyObject_Call_Prepend
    @     0x5612c5569fb6 slot_tp_call

3:,env:
(base) root@devpod2:/workspace/nvidia-cosmos/Depth-Anything-3-main# vidia-smi
Fri Jan 30 11:27:23 2026       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 550.144.03             Driver Version: 550.144.03     CUDA Version: 12.4     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA A100-PCIE-40GB          Off |   00000000:31:00.0 Off |                    0 |
| N/A   30C    P0             36W /  250W |    1443MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   1  NVIDIA A100-PCIE-40GB          Off |   00000000:4B:00.0 Off |                    0 |
| N/A   29C    P0             32W /  250W |       4MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   2  NVIDIA A100-PCIE-40GB          Off |   00000000:B1:00.0 Off |                    0 |
| N/A   29C    P0             32W /  250W |       4MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   3  NVIDIA A100-PCIE-40GB          Off |   00000000:CA:00.0 Off |                    0 |
| N/A   30C    P0             36W /  250W |       4MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
                                                                                         
+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
